website/source/docs/builders/triton.html.md

```diff
@@ -0,0 +1,65 @@
+---
+description: |
+    The `triton` Packer builder is able to create new images for use with Triton. These images can be used with both the Joyent public cloud (which is powered by Triton) as well with private Triton installations. This builder uses the Triton Cloud API to create images. The builder creates and launches a temporary VM based on a specified source image, runs any provisioning necessary, uses the Triton "VM to image" functionality to create a reusable image and finally destroys the temporary VM. This reusable image can then be used to launch new VM's.
+page_title: Triton Builder
+...
+
+# Triton Builder
+
+Type: `triton`
+
+The `triton` Packer builder is able to create new images for use with Triton. These images can be used with both the [Joyent public cloud](https://www.joyent.com/) (which is powered by Triton) as well with private [Triton](https://github.com/joyent/triton) installations. This builder uses the Triton Cloud API to create images. The builder creates and launches a temporary VM based on a specified source image, runs any provisioning necessary, uses the Triton "VM to image" functionality to create a reusable image and finally destroys the temporary VM. This reusable image can then be used to launch new VM's.
+
+The builder does *not* manage images. Once it creates an image, it is up to you to use it or delete it.
+
+## Configuration Reference
+
+There are many configuration options available for the builder. They are segmented below into two categories: required and optional parameters.
+
+In addition to the options listed here, a [communicator](/docs/templates/communicator.html) can be configured for this builder.
+
+### Required:
+
+-   `triton_account` (string) - The username of the Triton account to use when using the Triton Cloud API.
+-   `triton_key_id` (string) - The fingerprint of the public key of the SSH key pair to use for authentication against Triton.
+-   `triton_key_material` (string) - The path to the private key of the SSH key pair associated with the Triton account to be used. For example `~/.ssh/id_rsa`.
+ 
+ -   `source_machine_image` (string) - The UUID of the image to base the new image on. On the Joyent public cloud this could for example be `70e3ae72-96b6-11e6-9056-9737fd4d0764` for version 16.3.1 of the 64bit SmartOS base image.
```

Comment:
I searched for some sort of style guide for the documentation but I couldn't find any. Is there a guide somewhere or a rule set? Looking at `amazon-ebs.html.md` seemed to indicate some sort of 80 column limit? Is adhering to a 80 column width enough? Let me know how you guys would like it formatted and i'll fix it up!